pkg/aws/v1alpha3/awscontrolplane/validate_awscontrolplane.go

```diff
@@ -230,22 +230,6 @@ func (v *Validator) InstanceTypeValid(awsControlPlane infrastructurev1alpha3.AWS
 	return nil
 }
 
-func (v *Validator) isValidMasterAvailabilityZones(availabilityZones []string) bool {
```

Comment:
Since the mechanism is the same, I moved this to a central function for both workers and controlplanes.